src/SCRIPTS/BF/ui_init.lua

```diff
@@ -18,6 +22,19 @@ local function init()
         data_init = nil
         apiVersionReceived = true
         collectgarbage()
+    elseif not craftName then
+        lcd.drawText(6, radio.yMinLimit, "Waiting for craft name")
+        getCraftName = getCraftName or assert(loadScript("craft_name.lua"))()
+        craftNameReceived = getCraftName()
+        if craftNameReceived then
+            getCraftName = nil
+            vtxTables = loadScript("/BF/VTX/"..craftName..".lua")
+            if vtxTables and vtxTables() then
+               vtxTablesReceived = true
+               vtxTables = nil
+            end
+            collectgarbage()
+         end
```

Comment:
I think downloading ```craftName``` should be conditional to ```apiVersion >= 1.042``` since it's only needed for firmwares that support vtx tables. This will save some time at init for older firmwares.

```craftNameReceived``` should be made local to this file so that it's garbage collected when init is done.